tests/impl.cpp

```diff
@@ -3789,7 +3789,20 @@ result_t test_mm_cmpnlt_sd(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_cmpord_pd(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const double *_a = (const double *) impl.mTestFloatPointer1;
+    const double *_b = (const double *) impl.mTestFloatPointer2;
+    __m128d a = _mm_load_pd(_a);
+    __m128d b = _mm_load_pd(_b);
+
+    float result[2];
+
+    for (uint32_t i = 0; i < 2; i++) {
+        result[i] = compord(_a[i], _b[i]);
```

Comment:
The `compord` should have a `double` version instead of using the `float` version.

Reply:
new version has been added.